### PR TITLE
Fix sonar warnings Part 2

### PR DESCRIPTION
**Pull request by lilian-houdelet**

### CODE REVIEW

**Review by dbraquart — COMMENTED** [review]
code: ok
tests: ok
Need a 2nd reviewer to validate the @lazy annotation

**Review by Tristan-WorkGH — APPROVED** [review]
OK for the `@Lazy` part.

File: src/main/java/org/gridsuite/study/server/service/NetworkModificationTreeService.java [hunk 1]
```diff
@@ -572,7 +563,7 @@ public String getSuffixedNodeName(UUID studyUuid, String nodeName) {
         return uniqueName;
     }
 
-    @Transactional(readOnly = false)
+    @Transactional
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/study/server/service/NetworkModificationTreeService.java:566]: ?

**Review by SlimaneAmar — APPROVED** [review]

File: src/main/java/org/gridsuite/study/server/service/NetworkModificationTreeService.java (hunk 1 above)
Comment (lilian-houdelet) [reply src/main/java/org/gridsuite/study/server/service/NetworkModificationTreeService.java:566]: default value is false
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/transaction/annotation/Transactional.html

**Pull request merged**